### ISSUE TITLE
Can't connect Grafana/Prometus and sun app

### ISSUE THREAD

**Issue by GaryWayneSmith** [ginoledesma/sunpower-pvs-exporter#7]
I think I have everything configured correctly.

I have Prometheus and Grafana set to start on boot.  Running curl http://localhost:9110/ returns the data as expected.  When to into the new install of Grafana and I add Prometheus as a data source I click test and get no errors.  When I look into the query and run it I can see that it returns the same results as curl under cli.

The data, however, still remain to have zero data and under the inspect I get e.data.data is undefined under the error tab.  

What am I missing?

**Comment by rome1983** [comment]
When you setup data source do you see this after?  Data source is working
I had a similar issue, come to find out I was using the wrong port. using 9110 instead of 9090

**Comment by GaryWayneSmith** [comment]
9090 gives me bad gateway.  9110 saves/tests just fine.  When I go to one of the widgets and then do query inspector, query then refresh query I get results.  On the error tab it says e.data.data is undefined (not sure if that means anything).  Under stats, queries =1, total rows = 0.  I'm not sure if the data I'm looking at is correct though.


![sunpower_data_test](https://user-images.githubusercontent.com/9663671/95765978-75a75f00-0c67-11eb-85ff-f2fee3f33144.PNG)

**Comment by rome1983** [comment]
Unless you changed the default port Prometheus creates its database on port 9090. What I see on the right is incorrect because that’s not database format from prometheus. Keep in mind the default configs are outdated with new versions of grafana.

**Comment by GaryWayneSmith** [comment]
Makes sense.  Is there a particular version of grafana I should be running?

**Comment by rome1983** [comment]
I’m running the latest. Check out the fork 
https://github.com/xlukose/sunpower-pvs-exporter

His configs are updated
